SDK-3160 ios foreground changes (#4382)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/braze_currents/event_glossary/message_engagement_events.md

Push, iOS, Sends
{% endapitags %}

> Please note, this event is now deprecated on our [Swift SDK](https://github.com/braze-inc/braze-swift-sdk).

Reviewer: ```suggestion
Please note, this event is deprecated and not supported by our [Swift SDK](https://github.com/braze-inc/braze-swift-sdk).
```

> This event only occurs if a push was sent while the iOS app was in the foreground and your developers have integrated the Obj-C iOS SDK for foreground push handling detailed [here]({{site.baseurl}}/developer_guide/platform_integration_guides/ios/push_notifications/integration/#ios-10).

Reviewer: change to deprecation notice on obj-c
